homeassistant/components/twinkly/light.py

```diff
@@ -46,7 +46,7 @@ async def async_setup_entry(
     config_entry: ConfigEntry,
     async_add_entities: AddEntitiesCallback,
 ) -> None:
-    """Setups an entity from a config entry (UI config flow)."""
+    """Set up an entity from a config entry (UI config flow)."""
```

Comment:
Setup is a noun, but this should be a verb phrase. And while `Sets up` would be a natural change, the linter insists on `Set`